tests/migrations/test_migrations.py

```diff
@@ -397,3 +397,21 @@ def test_migrations_to_ba8234fe9eb5(alembic_runner: MigrationContext, alembic_en
     inspector = sqlalchemy.inspect(alembic_engine)
 
     assert "phone_ext" in set([c["name"] for c in inspector.get_columns("contact_info")])
+
+
+def test_migrations_to_6babc6109a5a(alembic_runner: MigrationContext, alembic_engine: Engine):
```

Comment:
these should probably test for the length limit added; check to see if the inspector is able to give back the column type, and if the type contains the length limit.